shopfloor_reception/models/shopfloor_menu.py

```diff
@@ -17,13 +18,25 @@ class ShopfloorMenu(models.Model):
         default=False,
         help=AUTO_POST_LINE,
     )
-    auto_post_line_is_possible = fields.Boolean(
-        compute="_compute_auto_post_line_is_possible"
+    auto_post_line_is_possible = fields.Boolean(compute="_compute_scenario_options")
+
+    select_move_by_click = fields.Boolean(
```

Comment:
This is already possible by using manual-select widget instead of a simple list.
Please, have a look at `checkout`.

Reply:
This would be needed the configure, if you can select a move without scanning the barcode.
I know that there is already the manual-select widget.

But in general. I wanted to add more possibilities to the item-detail-card, to make it easier to overwrite it by a other module without, overwriting the template. Because as it is right now, i think an overwrite of an existing scenario is not easy to implement or not nice.

Like you can see it here. 
https://github.com/OCA/wms/blob/c11fb21b1cd7ef5d125c13df82c248b4fc425ec2/shopfloor_checkout_package_measurement_mobile/static/src/scenario/checkout_package_measurement.js#L15

In our case we need a button on right sight of the item-detail-card at reception for only specific marked products. 

In fact the best would be to change, how/where the scenario template str is generated. I think it would be nice to move this out of javascript .